### PR TITLE
CB-4767 generate domains using deployment manager

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: webapp/packages/core-ui/src/Form/FormState.ts [hunk 1]
```diff
@@ -178,6 +183,12 @@ export class FormState<TState> implements IFormState<TState> {
     }
   }
 
+  reset(): void {
```
Comment (sergeyteleshev) [inline webapp/packages/core-ui/src/Form/FormState.ts:186]: resetParts?

Comment (sergeyteleshev) [inline webapp/packages/core-ui/src/Form/FormState.ts:186]: resetParts?

File: webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfoPartBootstrap.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class UserProfileFormInfoPartBootstrap extends Bootstrap {
       name: 'plugin_user_profile_info',
       order: 1,
       panel: () => UserProfileFormInfo,
-      stateGetter: props => () => props.formState.dataContext.get(DATA_CONTEXT_USER_PROFILE_FORM_INFO_PART),
+      stateGetter: props => () => props.formState.getPart(DATA_CONTEXT_USER_PROFILE_FORM_INFO_PART),
```
Comment (sergeyteleshev) [inline webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfoPartBootstrap.ts:28]: should we do same thing with getPart in similar cases all over the app?

**Review by devnaumov — APPROVED** [review]

File: webapp/packages/core-ui/src/Form/FormState.ts (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core-ui/src/Form/FormState.ts:186]: just reset, cuz it's like `form.reset()` we don't need to know details of function implementation cuz we actually want to reset the whole form

File: webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfoPartBootstrap.ts (hunk 2 above)
Comment (Wroud) [reply webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfoPartBootstrap.ts:28]: this is only case by now, but we will rework how we access parts in other places later (probably we will replace data context with .getPart)

**Review by sergeyteleshev — APPROVED** [review]

File: server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/CBServerConfigurationController.java [hunk 3]
```diff
@@ -363,6 +364,11 @@ private void writeRuntimeConfig(Path runtimeConfigPath, Map<String, Object> conf
         }
     }
 
+
+    public void saveCommonConfig(@NotNull SMCredentialsProvider credentialsProvider, @Nullable String newPublicUrl) throws DBException {
```
Comment (alexander-skoblikov) [inline server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/CBServerConfigurationController.java]: update server url here

File: server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java [hunk 4]
```diff
@@ -1815,7 +1815,8 @@ private void invalidateUserTokens(String smToken) throws DBCException {
         }
     }
 
-    private void invalidateAllUserTokens(String userId) throws DBCException {
+    @Override
+    public void invalidateAllUserTokens(@NotNull String userId) throws DBCException {
```
Comment (alexander-skoblikov) [inline server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java]: invalidate all users

**Pull request merged**